### PR TITLE
feat: set more consumer metadata on message processing

### PR DESCRIPTION
**Pull request by btkostner**
## Related Ticket(s)

N/A

## Checklist

- [X] Code conforms to the [Elixir Styleguide](https://github.com/christopheradams/elixir_style_guide)

## Problem

Most of our consumer code sets these fields for every message.

## Details

Simplify our use case so everyone gets the benefit.

### CODE REVIEW

File: lib/kafee/consumer/message.ex [hunk 1]
```diff
@@ -152,12 +152,21 @@ defmodule Kafee.Consumer.Message do
...
     end
+
+    [
+      :key,
+      :topic,
+      :partition,
+      :offset
+    ]
+    |> Enum.map(fn key -> {key, Map.get(message, key)} end)
+    |> Logger.metadata()
```
Comment (msutkowski) [inline lib/kafee/consumer/message.ex:170]: I was thinking about this about a month ago... what do you think about adopting a similar format to the otel spec so we're kinda normalized across logs and traces? See the `messaging.*` and `messaging.kafka.*` for reference.

image

Comment (btkostner) [reply lib/kafee/consumer/message.ex:170]: Seems like a solid idea. We already set those on traces but they should be the same keys for the logs.

Comment (msutkowski) [reply lib/kafee/consumer/message.ex:170]: Awesome. I really think that'll make things easier long term as there's kinda just one pattern to remember when searching for things and we can always just point to the opentelemetry docs and standards 🎉

Comment (btkostner) [reply lib/kafee/consumer/message.ex]: Updated. Let me know if this works for you.

**Review by msutkowski — APPROVED** [review]

**Pull request merged**